R/rmarkdown.R

```diff
@@ -38,10 +38,10 @@ render_rmarkdown <- function(pkg, input, output, ..., copy_images = TRUE, quiet
       )
     ),
     error = function(cnd) {
-      rule("RMarkdown error")
-      cat(gsub("\r", "", cnd$stderr, fixed = TRUE))
```

Comment:
Looks like you're no longer displaying the output from stderr, which, IIRC, is pretty important.